conf/machine/orangepi-i96.conf

```diff
@@ -0,0 +1,40 @@
+#@TYPE: Machine
+#@NAME: OrangePi i96 machine
+#@DESCRIPTION: OrangePi i96 machine
+
+require conf/machine/include/tune-cortexa5.inc
+
+PREFERRED_PROVIDER_virtual/xserver ?= "xserver-xorg"
+
+XSERVER ?= "xserver-xorg \
+            mesa-driver-swrast \
+            xf86-input-evdev \
+            xf86-video-fbdev"
```

Comment:
```
            xf86-video-fbdev \
"
```

Reply:
Can we change 'fbdev' to 'modesetting'?

Reply:
This thing is running with a v3.10 vendor kernel. I haven't checked explicitly (i96 not having easy access to video out) but I'd be surprised if it has a working KMS implemention.